extensions/functions_rounding.yaml

```diff
@@ -268,3 +268,43 @@ scalar_functions:
               AWAY_FROM_ZERO, TIE_DOWN, TIE_UP, TIE_TOWARDS_ZERO, TIE_TO_ODD ]
         nullability: DECLARED_OUTPUT
         return: fp64?
+      - args:
```

Comment:
For all other decimal functionality we have placed them in `_decimal.yaml` files.  Not sure if we want to have just this one function in a file by itself though.